docs/user/autoyast/reference.md

```diff
@@ -0,0 +1,402 @@
+# AutoYaST compatibility reference
+
+Let's describe which sections and elements from an AutoYaST profile are (or will be) supported in
+Agama. In some cases, you might find a table with the following columns:
+
+- AutoYaST: name of the AutoYaST element.
+- Supported: whether it is (or will be) supported. The meaning of each value is:
+  - Yes: fully supported.
+  - Planned: not supported yet, but there are plans to support it.
+  - Undecided: no decision about whether it should be supported.
+  - No: there are no plans for supporting that element.
+- Agama: name of the Agama element.
+- Comment: any comment or reason about the element.
+
+## add-on
+
+This section is not supported.
+
+## audit-laf
+
+This section is not supported.
+
+## auth-client
+
+This section is not supported.
+
+## configuration_management
+
+This section is not supported.
+
+## deploy_image
+
+This section is not supported.
+
+## dhcp-server
+
+This section is not supported.
+
+## dns-server
+
+This section is not supported.
+
+## fcoe-client
+
+This section is not supported.
+
+## files
+
+There are plans to support this section in the future.
```

Comment:
This section in the middle of all those "this section is not supported" is somehow hard to spot. Maybe we could use bold for "not" and for "future". That would make it easier to do a quick scan.

Reply:
And for "supported" in the positive case.

Reply:
Yes, that's true. But idea is to put all the unsupported sections in a single section. However, there are tons of things to do first.